### DISCUSSION TITLE
pipxu is fantastic, just works and is fast and efficient!

### DISCUSSION THREAD

**Discussion by idvorkin** [bulletmark/pipxu#10] [General]
Feel free to close this "issue" once you've read. I wanted to say thank you for this wonderful package, it just works, is super fast, and disk space efficient. 

Since I've upgraded to python 3.12, I've had to start using pipx, which I love in theory, but being based on pip is super slow, pipxu is exactly what the community needs

Thank you again!

**Comment by bulletmark** [comment]
Thanks, yes I personally use it all the time and really just made it for myself but glad to see others using it. As a developer, `pipxu` offers some nice features that `pipx` does not have (debug command which I use frequently, "." shortcut, shows edit path in list, etc).

The speed all comes from the massive improvement `uv` gives over `venv` + `pip`. I haven't used those two since `uv` appeared.
